README.md

```diff
@@ -27,149 +28,171 @@ First, install the library via [npm](https://www.npmjs.com/get-npm):
 ```bash
 npm install @yext/analytics
 ```
+### Initialize Analytics Reporter
 
-Next, import and initialize the library in your application.  Yext currently has different analytics reporting features between Search and Pages and so they have slightly different interfaces for working with them.  There is also a combined interface that you can use when you are building a Search experience entirely on Pages (e.g. a Locator or a Help Site).
-
-### Search Analytics
-
+Next, import and initialize the library in your application. When initializing your analytics reporter, you only need to provide an API Key that has access to the Events API. Other attributes such as your business ID will be automatically inferred. You can acquire this API key in the developer console of the Yext Platform.
 ```ts
-import { provideSearchAnalytics } from '@yext/analytics'; // can also be imported as provideAnalytics
+import { Analytics } from "@yext/analytics";
 
-const searchAnalytics = provideSearchAnalytics({
-  experienceKey: '<your experience key>',
-  experienceVersion: 'PRODUCTION',
-  businessId: 123456, // this comes from the url of your Yext account
-});
+// Root analytics service with no defaults.
+const rootAnalytics = new Analytics("MY_API_KEY"); 
 ```
+In many cases, you might need to repeatedly specify the same properties, such as a Pages site ID or Chat bot ID. Yext Analytics allows you to avoid having to repeatedly specify the same code by allowing you to set **default values**.
 
-To use the library with Node, use the following import instead:
+You can add a `.with()` method to the root analytics service you initialized, which returns a new analytics object with the specified JSON merged on top of the existing defaults.
 ```ts
-const { provideSearchAnalytics } = require('@yext/analytics');
-``` 
+import { Analytics } from "@yext/analytics";
 
-Now that the search analytics reporter is initialized, let's fire off an event:
+// Root analytics service with no defaults.
+const rootAnalytics = new Analytics("MY_API_KEY"); 
```

Comment:
Can you do this? I thought you would need to pass it in like an (AnalyticsConfig) object. 

```
const rootAnalytics = new Analytics( { key: "MY_API_KEY" } );

Reply:
Ah, you're right.